woocommerce/woocommerce
# Add/a2p create tax rates crud api core tests

### All Submissions:






### Changes proposed in this Pull Request:
Adding CRUD tests for the tax rates and tax classes to ensure we have better test coverage



Closes 370-gh-woocommerce/woocommerce-quality and 369-gh-woocommerce/woocommerce-quality.

### How to test the changes in this Pull Request:

1. Pull down the code
2. Navigate to `woocommerce/plugins/woocommerce`
3. Execute tests with `USE_WP_ENV=1 pnpm playwright test --config=tests/api-core-tests/playwright.config.js tests/api-core-tests/tests/taxes`
4. The 15 taxes tests should execute and pass successfully

### Other information:




### FOR PR REVIEWER ONLY:
